crates/bevy_ui/src/layout/mod.rs

```diff
@@ -177,7 +177,7 @@ pub fn ui_layout_system(
                 Some(camera_entity) => {
                     let Ok((_, camera)) = cameras.get(camera_entity) else {
                         warn!(
-                            "TargetCamera (of root UI node {entity:?}) is pointing to a camera {:?} which doesn't exist",
+                            "TargetCamera (of root UI node {entity:?}) is pointing to a camera {} which doesn't exist",
```

Comment:
```suggestion
                            "TargetCamera (of root UI node {entity}) is pointing to a camera {} which doesn't exist",
```
I guess there's a leftover `:?` there, right?